### PR TITLE
feat: add an option to configure log-pilot to ignore or/and only handle containers from specified namespaces

### PR DESCRIPTION
**Pull request by lichuan0620**
**What this PR does / why we need it**:

In order to support dedicated ES for system log, it is necessary for containers from different namespaces to be outputted to different ES. This PR adds an option to configure log-pilot to ignore or/and only handle containers from certain namespaces via command argument. 

By default, all namespaces are watched, so we won't have to update existing configuration for other clients.

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to https://github.com/caicloud/logging-admin/issues/179

**Special notes for your reviewer**:

/cc @caitong93 @supereagle 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by lichuan0620** [comment]
Just a draft. Not tested.

@caitong93 PTAL

Current logic:
1. have a whitelist and blacklist of namespaces
2. Before running `filebeatConfigurer.OnAdd` and `filebeatConfigurer.OnDestroy`, judge by namespace whether or not the container should be handled
3. If not, the two function return without doing anything; if yes, proceed as usual.

Any feedback?

**Comment by caitong93** [comment]
@lichuan0620 
You can do filtering at https://github.com/caicloud/log-pilot/blob/master/pilot/discovery/discovery.go#L252

**Comment by lichuan0620** [comment]
@caitong93 PTAL

File: pilot/configurer/filebeat/filebeat.go [hunk 1]
```diff
@@ -372,3 +376,22 @@ func (c *filebeatConfigurer) OnDestroy(ev *configurer.ContainerDestroyEvent) err
 func (c *filebeatConfigurer) Stop() {
 	close(c.closeCh)
 }
+
+func (c *filebeatConfigurer) IsResponsible(namespace string) bool {
```
Comment (caitong93) [inline pilot/configurer/filebeat/filebeat.go]: This can be moved to discovery

**Review by caitong93 — CHANGES_REQUESTED** [review]

**Review by caitong93 — APPROVED** [review]

**Comment by lichuan0620** [comment]
tested, PTAL
@caitong93 @supereagle

**Pull request merged**